src/test/groovy/com/pszymczyk/consul/ConsulStarterTest.groovy

```diff
@@ -109,4 +113,38 @@ class ConsulStarterTest extends Specification {
         then:
         consulClient.getAgentChecks().value.isEmpty()
     }
+
+    def "should apply new custom config when restart with same config dir"() {
```

Comment:
I was not familiar with Groovy and Spock, so I mistook `setupSpec()` and JUnit `@Before` for the same. I agree that there was a problem with my test code.

And, the purpose of the test is to make sure that the options are changed properly, so I change my test code to comparing option value that acquired by consul client.